index.html

```diff
@@ -1,133 +1,111 @@
+                    world-leading companies and trailblazers share the latest thought leadership about the
+                    progression and future of connectivity. And it’s the best place for networking opportunities <br>
+                    with mobile and tech industry influencers.</p>
+                <h3>Barcelona Spain <br>
+                    28 February – 3 March 2022</h3>
+            </div>
         </section>
     </header>
     <section class="secondSection">
         <h2>Main Program</h2>
-        <div class="mainContainer">
-            <div class="spaceSpeacker">
-                <img  class="icon" src="./img/icon1.png" alt="speackersicon">
+        <div class="mainContainer deskContainer">
+            <div class="spaceSpeacker spacedesk">
+                <img class="icon" src="./img/icon1.png" alt="speackersicon">
                 <span class="programTitle">Speackers</span>
-            <p class="programP">It’s where world-leading companies and trailblazers share the latest thought leadership.
+                <p class="programP">It’s where world-leading companies and trailblazers share the latest thought
+                    leadership.
             </div>
-            <div  class="spaceSpeacker">
+            <div class="spaceSpeacker  spacedesk">
                 <img class="icon" src="./img/exhibitionicon.png" alt="exhibitionicon">
                 <span class="programTitle">Exhibition</span>
-            <p class="programP"> It is the place to be seen, exhibit ground-breaking products and technologies, <br> 
-                 and make remarkable connections.</p>
+                <p class="programP"> It is the place to be seen, exhibit ground-breaking products and technologies,
+                    and make remarkable connections.</p>
             </div>
-            <div  class="spaceSpeacker">
+            <div class="spaceSpeacker  spacedesk">
                 <img class="icon" src="./img/forumicon.png" alt="">
                 <span class="programTitle">Forum</span>
-            <p class="programP">Get ready to explore, debate and connect with  creators, and innovators in the industry. </p>
+                <p class="programP">Get ready to explore, debate and connect with creators, and innovators in the
+                    industry. </p>
             </div>
-            <div  class="spaceSpeacker">
+            <div class="spaceSpeacker  spacedesk">
                 <img class="icon" src="./img/workshopicon.png" alt="">
                 <span class="programTitle">Workshop</span>
-            <p class="programP">Whether you’re launching a new product or looking for partners to bring future innovations to life.</p>
+                <p class="programP">Whether you’re launching a new product or looking for partners to bring future
+                    innovations to life.</p>
             </div>
         </div>
         <button class="programbtn">Become an Exhibitor</button>
-
     </section>
```

Comment:
- [ ] On the main program's section of the page, the items are aligned in a row whereas it is supposed to be aligned in a column on desktop view and a row on mobile view.
Your design👇🏽
![Capture4](https://user-images.githubusercontent.com/61285247/133878981-eb828fd9-2896-4d04-9a53-76363f5747bd.PNG)
Please kindly fix that.

- [ ] On desktop view there should be no button on the main program's section. Please check the original design and kindly fix that.

Reply:
Thanks for your review but I checked many times on the computer, on mobile, and on Github pages and the navbar is working correctly same as the link for the about me page.
The main section and the featured speaker's section are aligned in the column as well.
I really don't understand can you check, please.

Reply:
- [ ] In the main program's section on desktop view there is no button but plain text. You can check the original design to see for yourself.
- [ ] Your page has a small overflow. Please kindly fix that.